### PR TITLE
Add 2019 December Public Board Meeting minutes

### PR DESCRIPTION
**Pull request by cjfields**
These need to be approved at the next public board meeting prior to merging, see issue #68 for an example.

### CODE REVIEW

**Comment by peterjc** [comment]
Tip: Click on the "Files changed" tap, then top right "Review changes", can add comment text and select approve.

**Comment by hlapp** [comment]
Approval can really only happen at a subsequent public Board meeting, FYI.

**Comment by cjfields** [comment]
@peterjc @hlapp related to this, should we add official reviewing for pull requests?  This would be an informal form of approval that could take place prior to an official public board vote.

**Comment by peterjc** [comment]
You mean enabling GitHub settings to require GitHub based approval before allowing a merge? Good idea.

On top of that, some changes (like approving minutes and changes to bylaws) have a further hurdle for approval.

**Comment by cjfields** [comment]
> You mean enabling GitHub settings to require GitHub based approval before allowing a merge? Good idea.
> 
> On top of that, some changes (like approving minutes and changes to bylaws) have a further hurdle for approval.

Yes, this.

**Comment by cjfields** [comment]
Also, see #72 for ways to set this up and a place to discuss this a bit more, as we could possibly all set up the master as a protected branch (which I think could prevent accidental commits or merges).

**Comment by cjfields** [comment]
@peterjc @hlapp I have a branch rule set up to prevent merges, including for admins. I arbitrarily required this needing four reviewers but we can have more or less.  Branch rule is here:

https://github.com/OBF/obf-docs/settings/branches

**Comment by peterjc** [comment]
I notice that neither this draft, nor indeed https://github.com/OBF/obf-docs/blob/master/minutes/2019-July.md has the actual date - just the year and month.

Should we clarify this was 17 December 2019?

**Comment by hlapp** [comment]
Indeed, that's an oversight. We should amend the July 2019 minutes with this as well. (Doing so shouldn't require another Public Board meeting approval because this would not alter the actual minutes.)

**Comment by cjfields** [comment]
Agreed.  Since master branch is now protected I don't think we can directly make commits w/o a pull request, but I can try with the July 2019 minutes.  I'll go back and check the date.

**Comment by hlapp** [comment]
I made #73 for the July 2019 minutes. BTW I also restored https://www.open-bio.org/2019/07/23/minutes2019-bosc/ to a prior version (and then changed to past tense) because it is being referred to for the agenda. The latest version there had lost that, as well as date and location 😔

**Comment by cjfields** [comment]
> I made #73 for the July 2019 minutes. BTW I also restored https://www.open-bio.org/2019/07/23/minutes2019-bosc/ to a prior version (and then changed to past tense) because it is being referred to for the agenda. The latest version there had lost that, as well as date and location 😔

Ah, thanks for the pull request.  I had tested out trying to push this change to master and protecting the branch worked wonderfully 😄 

```
11:18 $ git push
Counting objects: 4, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (3/3), done.
Writing objects: 100% (4/4), 361 bytes | 180.00 KiB/s, done.
Total 4 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: At least 4 approving reviews are required by reviewers with write access.
To github.com:obf/obf-docs
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:obf/obf-docs'
✘-1 ~/src/obf-docs/minutes [master ↑·1|✔]
```

**Comment by cjfields** [comment]
BTW, if we need to set up automatic review requests:

https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team

File: minutes/2019-Dec.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+## 2019-December Public Board Meeting
+
+**Present**: Hilmar Lapp, Peter Cock, Heather Wiencko, Chris Fields, Yo Yehudi, Nomi Harris, Bastian Greshake-Tzovaras
+
+**Absent**: Malvika Sharan
+
+*Agenda*: [https://github.com/OBF/org/issues/2](https://github.com/OBF/org/issues/2)
```
Comment (hlapp) [inline minutes/2019-Dec.md]:
```suggestion
*Agenda*: [OBF/org#2](https://github.com/OBF/org/issues/2)
```

File: minutes/2019-Dec.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+*   Nomi Harris: running for re-election, as At-Large Board member.
+
+Candidate statements:
+
+*   Peter: Likes direction that OBF is heading.  Would like to add explicit travel support for Bio* developers for conferences/hackathons.
+*   Hilmar: Greatly enjoyed time as president, one of the most rewarding things I’ve done.  Since elected President in 2011 have seen significant improvements and very happy with progress and long-term sustainability.  Travel fellowship program started since then, and will soon have an affiliate project policy in place.  Great to see new people come to flagship events, and know this will continue w/o me.
+*   Heather: Changing up how we’re reporting finances.  Very happy to take on the role of treasurer as this proceeds forward.
+*   Nomi, Chris - not much more to add to what Hilmar and Heather covered beyond that it’s been an incredibly positive experience contributing to OBF, and they want to continue doing so
+
+**Election Results:** [https://vote.heliosvoting.org/helios/e/obf-dec-2019](https://vote.heliosvoting.org/helios/e/obf-dec-2019) (all elected, no votes against)
```
Comment (hlapp) [inline minutes/2019-Dec.md]:
```suggestion
**Election Results:** <https://vote.heliosvoting.org/helios/e/obf-dec-2019> (all elected, no votes against)
```

File: minutes/2019-Dec.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## [Affiliated Project Status policy](https://github.com/OBF/obf-docs/pull/44). 
+
+The public RfC period closed with no additional change requests or comments. The draft is ready for a vote to approve, adopt, and ratify, both for the Board, and for the OBF membership.
+
+*   Peter: A question re: GSoC addition as a ‘grandfather project’.  This is in relation to the addition over the last 24 hrs by Michael Heuer.  
+    *   The change is actually fixing a broken link, and references to GSoC are not in relation to being a separate project (Peter’s misunderstanding was clarified). 
+*   No other objections or comments
+
+**Voting results: For: 7, Against: 0:** [https://vote.heliosvoting.org/helios/e/obf-affiliate-policy-board](https://vote.heliosvoting.org/helios/e/obf-affiliate-policy-board)
```
Comment (hlapp) [inline minutes/2019-Dec.md]:
```suggestion
**Voting results: For: 7, Against: 0:** <https://vote.heliosvoting.org/helios/e/obf-affiliate-policy-board>
```

File: minutes/2019-Dec.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Now can release ballot to membership for overall approval.   Election will close noon New Year’s Eve.
+
+*   Maybe extend to end of first full week in January (Jan 10)? 
+    *   Hilmar motions to approve, Yo and Chris second.  Approved by unanimous consent.
+    *   Approval from the membership needs quorum (10%).  We have approximately 270 members so need ~27 for quorum; can look at rolls to get more accurate count of members. 
+*   Hilmar - will recreate ballot for full OBF membership, try to release to go out by today
+
+## OBF-wide Code of Conduct policy: Status report
+
+*   Feedback is present on Github:[https://github.com/OBF/obf-docs/issues/67](https://github.com/OBF/obf-docs/issues/67)
```
Comment (hlapp) [inline minutes/2019-Dec.md]:
```suggestion
*   Feedback is present on Github: [OBF/obf-docs#67](https://github.com/OBF/obf-docs/issues/67)
```

File: minutes/2019-Dec.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+*   Hilmar - expresses gratitude on behalf of entire Board to Bastian, Yo, and Peter for working on this
+
+## Financial reporting
+
+*   Peter: update on current balances
+    *   ISCB escrow: Potentially $2000 overdrawn, but not sure what has been billed and what hasn’t.  Come 2020 we need to bring up to at least zero and ideally $1000 to be in good standing with ISCB.  
+    *   SPI: $84,498 - even w/ travel fellowships, sitting well over $80k even with travel fellowships.  In terms of commitments some is committed to fellowships but still very healthy. 
+*   How much is earmarked for fellowship from NESCent?  
+    *   Link to the original announcement: [https://www.open-bio.org/2016/03/01/obf-travel-fellowship-program/](https://www.open-bio.org/2016/03/01/obf-travel-fellowship-program/)
```
Comment (hlapp) [inline minutes/2019-Dec.md]:
```suggestion
    *   Link to the original announcement: <https://www.open-bio.org/2016/03/01/obf-travel-fellowship-program/>
```

File: minutes/2019-Dec.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+*   Hilmar - expresses gratitude on behalf of entire Board to Bastian, Yo, and Peter for working on this
+
+## Financial reporting
+
+*   Peter: update on current balances
+    *   ISCB escrow: Potentially $2000 overdrawn, but not sure what has been billed and what hasn’t.  Come 2020 we need to bring up to at least zero and ideally $1000 to be in good standing with ISCB.  
+    *   SPI: $84,498 - even w/ travel fellowships, sitting well over $80k even with travel fellowships.  In terms of commitments some is committed to fellowships but still very healthy. 
+*   How much is earmarked for fellowship from NESCent?  
+    *   Link to the original announcement: [https://www.open-bio.org/2016/03/01/obf-travel-fellowship-program/](https://www.open-bio.org/2016/03/01/obf-travel-fellowship-program/) 
+    *   Additional announcement including the NESCent donation: [https://www.open-bio.org/2016/04/18/phylosoc-supports-obf-travel-fellowships/](https://www.open-bio.org/2016/04/18/phylosoc-supports-obf-travel-fellowships/)
```
Comment (hlapp) [inline minutes/2019-Dec.md]:
```suggestion
    *   Additional announcement including the NESCent donation: <https://www.open-bio.org/2016/04/18/phylosoc-supports-obf-travel-fellowships/>
```

**Comment by cjfields** [comment]
@hlapp thanks for the edits, those were from the Google Doc plugin conversion.

**Comment by cjfields** [comment]
@OBF/board a little nudge, any additional reviews would be great!

**Comment by cjfields** [comment]
@HLWiencko @malvikasharan @yochannah let us know if you can review this (we can also reduce the # reviewers required if needed).  Thanks!

**Comment by hlapp** [comment]
> we can also reduce the # reviewers required if needed

Just as a reminder and as stated in the OP, public Board meeting minutes need to be formally approved at a subsequent public Board meeting. It doesn't matter how many do or do not approve them here on GitHub.

**Comment by cjfields** [comment]
@hlapp noted.  At the moment we have a minimum of 4 approvals from Board members (so @yochannah put this over the threshold).  We could change the merge restriction to something more suitable, I'll see what other options are available.

**Comment by cjfields** [comment]
Approved, merging (I completely missed this!)

**Comment by hlapp** [comment]
@cjfields we haven't had a public Board meeting to approve these, have we? Have you read the comments?

**Comment by cjfields** [comment]
@hlapp apologies, I read all but your last one (which was the most important!).  I'll try reverting.

**Comment by cjfields** [comment]
@hlapp see #85.  Ironically, we now need additional reviews for the _reversion_ of this pull request.  

Will tag this and additional pulls requiring public board approval going forward.

**Comment by peterjc** [comment]
The github tag is a good idea 👍

**Pull request merged**